scripts/populate_conferences.py

```diff
@@ -1,1034 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Do you know if there is an equivalent Django management command to replace this script? I know we can make meetings locally using the admin app but this could be kinda nice for devs to quickly populate a bunch of meetings? But also if this script is super broken/out of date it's probably not worth the trouble.

Reply:
I tried fixing it first! It was though so super broken (and had clearly not been updated in quite awhile) that it would probably be way easier to just write a new one if we needed it than to try to fix this one.